src/native/rollup.config.defines.js

```diff
@@ -0,0 +1,40 @@
+//! Licensed to the .NET Foundation under one or more agreements.
+//! The .NET Foundation licenses this file to you under the MIT license.
+
+import gitCommitInfo from "git-commit-info";
+
+export const configuration = process.env.Configuration || "Debug";
+export const productVersion = process.env.ProductVersion || "10.0.0-dev";
```

Comment:
This seems particularly fragile. Can we move this somewhere else?

Reply:
That is an excellent question. Summoning @jkoritzinsky who more than likely knows where this moniker comes from and can suggest how to create some sanitfy between our MSBuild and CMake.

Reply:
Converted to issue https://github.com/dotnet/runtime/issues/120188